docs-next/pages/guides/cli.mdx

```diff
@@ -2,81 +2,184 @@ import { Markdown } from '../../components/Page';
-Keystone can be run in two different modes; **dev** and **production**.
+- Generate the files required by Keystone's APIs and Admin UI
+- Generate and apply database migrations based on your Keystone Schema
+- Start the local development server, which hosts the API and Admin UI
 
-Note: All the **run** commands expect to find a module called `keystone.ts` with a default export which returns a system configuration `config()` from `@keystone-next/keystone/schema`.
-See the [System Configuration API](../apis/config) for details on how to configure a Keystone system.
+?> Keystone does not currently watch your local files for changes. If you update the config, schema or any other files in your keystone project you'll need to restart the server.
 
-#### keystone-next dev (default)
+### About database migrations
 
-In **dev** mode, Keystone will start the dev server and update your database is a different way depending on `db.useMigrations`:
+In development, Keystone will migrate the structure of your database for you in one of two ways depending on how you have configured `db.useMigrations`:
 
 - If `db.useMigrations` is `false` (the default), Keystone will use Prisma Migrate to update your database so that it matches your schema. It may lose data while updating your database so you should only use this mode in initial development.
 - If `db.useMigrations` is `true`, Keystone will use Prisma Migrate to apply any existing migrations and prompt you to create a migration if there was a change to your schema.
 
-#### keystone-next start
+When you are using migrations, the recommended workflow is to have `keystone-next dev` generate the migrations for you and apply them automatically in development.
+
+Commit the migration files to source control, then when you are hosting your app in production, use the `keystone-next prisma` command (see below) to deploy your migrations before starting Keystone.
+
+!> We strongly recommend enabling migrations if you are going to run your app in production. Not doing so risks data loss.
+
+### Resetting the database
+
+From time to time, in development you may want to reset the database and recreate it from scratch. You can do this by passing the `--reset-db` flag:
+
+```bash
+$ keystone-next dev --reset-db
+```
+
+x> Doing this will destroy your database, including all data
+
+This is mainly useful early in a project's development lifecycle, when you want to test database initialisation scripts or create a fresh set of test data.
+
+## postinstall
+
+```bash
+$ keystone-next postinstall
+```
+
+Keystone generates several files that your app may depend on, including the Node.js API and TypeScript definitions.
+
+While they are generated by Keystone's `dev` command, we strongly recomment you add the `postinstall` command to your project's `package.json` to ensure the files are always present and up to date; otherwise you may see intermittent errors in your Code Editor and CI workflows.
```

Comment:
```suggestion
While they are generated by Keystone's `dev` command, we strongly recomment you add the `postinstall` command to your project's `package.json` to ensure the files are always present and up to date; otherwise you may see intermittent errors in your code editor and CI workflows.
```